portfolio/src/main/webapp/index.html

```diff
@@ -81,8 +80,8 @@ <h4 class="my-3">To Inform</h4>
             </span>
             <h4 class="my-3">To Support</h4>
             <p class="text-muted">
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima maxime quam architecto quo inventore
-              harum ex magni, dicta impedit.
+              Support for all groups is key. We are here to provide support to all underrepresented groups, from black
+              women to LGBT+ groups.
```

Comment:
Same thing with being more general here. We want to provide resources and support for the black community, as well as provide a safe space for underrepresented groups.

super nit: add LGBTQ+ to be more inclusive

Reply:
thanks for catching that, you're absolutely right.